data/json/tool_qualities.json

```diff
@@ -211,5 +211,25 @@
     "name": "chromatography",
     "//": "Think of this as 'fine separating'. Chromatography represents a number of precise separation techniques",
     "//": "useful in both analysis and purification."
+  },
+  {
+    "type": "tool_quality",
+    "id": "VICE",
```

Comment:
This tool quality isn't used anywhere.

Reply:
Not yet. I'm currently looking at implementing it. Perhaps even removing it, I'm not so sure myself.